vignettes/sablefish_example.Rmd

```diff
@@ -0,0 +1,175 @@
+---
+title: "Working through sablefish PacFIN data"
+author: "Kelli F. Johnson"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Working through PacFIN data can be complex. This examples guides you through working up the data for sablefish. The fish ticket data contains information on catches and is labeled `catch.pacfin` within this vignette. The biological data that contains information on lengths and ages is labeled `bds.pacfin`. These names are standard for every data set that is downloaded and saved from PacFIN using `PacFIN.Utilities` extraction code.
+
+Before working up any of the data, a few items need to be clarified. Such as what gear types you would like to use and what bins should be used for the biological data. As well as which files you would like to load.
+
+```{r setup}
+# Species common name based on NWFSC survey data
+common_name <- "sablefish"
+# PacFIN species code
+species_code <- "SABL"
+
+# Files
+file_bds <- fs::path("..", "PacFIN.SABL.bds.11.Nov.2024.RData")
```

Comment:
I worked through an earlier version of this vignette using the file \\\\nwcfile\FRAM\Assessments\CurrentAssessments\sablefish_2025\data-raw\PacFIN.SABL.bds.17.Jul.2023.RData and the corresponding catch file, but I don't see anything like data_commercial_catch_expansions.csv on the network to work through the file.

However, the changes since a few weeks ago aren't that large and all look good so I don't need to run the code again.